### PR TITLE
Eq 1417 mwss content changes

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Updated clear content that meets our standards and accessibility for MWSS.
Card: https://trello.com/c/x6zJpTBI/1417-mwss-content-changes-m

### How to review 
- Check that `1_0005_updated.json` matches spec file and everything has matching q_codes to spec file.
- View: https://github.com/ONSdigital/eq-survey-runner/pull/1430/commits/7fe5bb2f18d3fc9212e44fffd265cb6be0c7fef3 to make it easier to compare the older version and the newer version.

### CODE REVIEW

File: data/en/1_0005_updated.json [hunk 1]
```diff
@@ -0,0 +1,2168 @@
...
+                    "id": "preview",
+                    "title": "Information you need",
+                    "content": [{
+                        "title": "Employee pay frequency",
+                        "description": "You will need to indicate how frequently employees are paid. For example weekly, fortnightly, calendar monthly, four weekly or five weekly."
+                    }, {
+                        "description": "<strong>You will need the following information for all of your selected pay patterns:</strong>"
+                    }],
+                    "questions": [{
+                        "question": "Number of paid employees",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json:68]: missing text `If the last week of the month is affected by holidays please use a morte representative week` at the top of this group

Comment (jonnyshaw89) [reply data/en/1_0005_updated.json:68]: There is also a type in the spec "please use a **morte** representative week"

File: data/en/1_0005_updated.json [hunk 2]
```diff
@@ -0,0 +1,2168 @@
...
+                            "list": [
+                                "Performance pay (for example productivity bonuses)",
+                                "Long service awards",
+                                "Appearance money (sporting professionals)"
+                            ]
+                        }, {
+                            "description": "Exclude:",
+                            "list": [
+                                "Trainees on government schemes",
+                                "Director’s fees",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: `Director’s fees` doesn't match the spec which has `Directors fees`

File: data/en/1_0005_updated.json [hunk 3]
```diff
@@ -0,0 +1,2168 @@
...
+                    "id": "pay-pattern-frequency-confirmation",
+                    "title": "Pay Pattern",
+                    "questions": [{
+                        "id": "pay-pattern-frequency-confirmation-question",
+                        "title": "Are these all of the pay patterns used?",
+                        "description": "{{ answers.pay_pattern|format_unordered_list }}",
+                        "type": "General",
+                        "answers": [{
+                            "id": "pay-pattern-frequency-confirmation-answer",
+                            "label": "Confirm that these are all of the pay patterns used to pay employees at {{respondent.trad_as_or_ru_name}}",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: This text is no longer in the spec

File: data/en/1_0005_updated.json [hunk 4]
```diff
@@ -0,0 +1,2168 @@
...
+                    ]
+                }
+            ],
+            "id": "pay-pattern",
+            "title": "Pay Pattern"
+        },
+        {
+            "blocks": [{
+                    "id": "weekly-pay-introduction",
+                    "title": "Weekly Pay",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json:305]: The spec has page and question numbers? but we don't have them any more?

File: data/en/1_0005_updated.json [hunk 5]
```diff
@@ -0,0 +1,2168 @@
...
+                            "q_code": "300w",
+                            "type": "TextArea",
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_TEXTAREA": "Enter a comment to continue."
+                                }
+                            }
+                        }],
+                        "id": "weekly-pay-significant-changes-other-specify-question",
+                        "title": "Please specify any other significant changes to the pay or number of <em>weekly paid employees</em>:",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: This text does not match the spec. The spec says `Please describe the changes to the <em>total pay or number</em> of <em>weekly</em> paid employees:`

File: data/en/1_0005_updated.json [hunk 6]
```diff
@@ -0,0 +1,2168 @@
...
+                    "id": "fortnightly-pay-significant-changes-pay-rates-increase",
+                    "questions": [{
+                        "answers": [{
+                                "id": "fortnightly-pay-significant-changes-pay-rates-increase-percent-increase-answer",
+                                "label": "Percentage increase of new pay rates",
+                                "mandatory": false,
+                                "q_code": "100f",
+                                "type": "Percentage",
+                                "max_value": {
+                                    "value": 100
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json:969]: Should this be limited to 100? a % increase can exceed 100.

File: data/en/1_0005_updated.json [hunk 7]
```diff
@@ -0,0 +1,2168 @@
...
+                                "block": "four-weekly-pay-introduction",
+                                "when": [{
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "condition": "equals",
+                                    "value": "No"
+                                }]
+                            }
+                        }
+                    ]
+
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: This text does not match the spec. the spec states "Please describe the changes to the <em>total pay or number</em> of <em>calendar monthly</em> paid employees:"

Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: This list doesn't match the spec. it should be 

bonuses or commissions 
overtime and shift allowance payments
advanced holiday pay
pay award arrears
voluntary employee contributions to, and annual profit from, profit related pay schemes

Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: this question text is wrong it should be Of the {{answers.five_weekly_total_gross|format_currency}} paid to five weekly paid employees in {{exercise.period_str}}, what other amounts were paid

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

File: data/en/1_0005_updated.json (hunk 6 above)
Comment (MIslam92) [reply data/en/1_0005_updated.json:969]: @jonnyshaw89 Yes technically your right but its a business requirement to have it capped at 100%

File: data/en/1_0005_updated.json (hunk 4 above)
Comment (shahi645) [reply data/en/1_0005_updated.json:305]: I think this was discussed before but we decided to remove section numbers

Comment (jonnyshaw89) [reply data/en/1_0005_updated.json:305]: @shahi645 therefore the specs should reflect this.

File: data/en/1_0005_updated.json [hunk 8]
```diff
@@ -0,0 +1,2172 @@
...
+                                "currency": "GBP",
+                                "decimal_places": 2,
+                                "max_value": {
+                                    "answer_id": "calendar-monthly-pay-gross-pay-answer",
+                                    "exclusive": true
+                                }
+                            }
+                        ],
+                        "id": "calendar-monthly-pay-breakdown-question",
+                        "title": "Of the <em>{{answers.five_weekly_total_gross|format_currency}}</em> paid to <em>five weekly</em> paid employees in {{exercise.period_str}}, what other amounts were paid:",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: I think this is wrong. you are piping `answers.five_weekly_total_gross` into a calendar monthly question?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by charlotte-hirst — COMMENTED** [review]
Checking " Please describe the changes to the total pay or number of fortnightly paid employees:' with sarah as its not quite worded as a question and Donna is checking validation on pay increase as to whether it should be mandatory to give an answer

File: data/en/1_0005_updated.json (hunk 8 above)
Comment (MIslam92) [reply data/en/1_0005_updated.json]: Lazy copying and pasting that is 🙃

**Comment by charlotte-hirst** [comment]
running to meeting but noticed on five weekly intro page says four weekly
"holiday pay, pay awards and bonuses paid to four weekly paid employees"

File: data/en/1_0005_updated.json [hunk 9]
```diff
@@ -0,0 +1,2172 @@
...
+                                    "value": "Yes"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "block": "four-weekly-pay-introduction",
+                                "when": [{
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "cond
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: this doesnt match spec should be " what other amounts were paid?"

Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: doesnt match spec should read "what other amounts were paid?"

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

File: data/en/1_0005_updated.json [hunk 10]
```diff
@@ -0,0 +1,2172 @@
...
+                                "when": [{
+                                    "id": "weekly-pay-significant-changes-paid-employees-answer",
+                                    "condition": "equals",
+                                    "value": "Yes"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "block": "fortnightly-pay-introduction",
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: can we not use the `goto` `group` now?

File: data/en/1_0005_updated.json [hunk 11]
```diff
@@ -0,0 +1,2172 @@
...
+                                    "value": "Yes"
+                                }]
+                            }
+                        },
+                        {
+                            "goto": {
+                                "block": "four-weekly-pay-introduction",
+                                "when": [{
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "con
```
Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: I think this title is wrong

Comment (jonnyshaw89) [inline data/en/1_0005_updated.json]: this references calendar monthly

Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: These labels can all be removed as it clashes with Hamish's work to make them add that label in by default, currently showing same label twice

File: data/en/1_0005_updated.json [hunk 12]
```diff
@@ -0,0 +1,2172 @@
...
+                    ]
+                },
+                {
+                    "id": "weekly-pay-significant-changes-options",
+                    "title": "Weekly Pay",
+                    "type": "Question",
+                    "questions": [{
+                        "answers": [{
+                            "id": "weekly-pay-significant-changes-options-answer",
+                            "label": "Select all that apply",
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: These labels can all be removed as it clashes with Hamish's work to make them add that label in by default, currently showing same label twice

File: data/en/1_0005_updated.json [hunk 13]
```diff
@@ -0,0 +1,2172 @@
...
+                    ]
+                },
+                {
+                    "id": "fortnightly-pay-significant-changes-options",
+                    "title": "Fortnightly Pay",
+                    "type": "Question",
+                    "questions": [{
+                        "answers": [{
+                            "id": "fortnightly-pay-significant-changes-options-answer",
+                            "label": "Select all that apply",
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: These labels can all be removed as it clashes with Hamish's work to make them add that label in by default, currently showing same label twice

File: data/en/1_0005_updated.json (hunk 11 above)
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: These labels can all be removed as it clashes with Hamish's work to make them add that label in by default, currently showing same label twice

Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: These labels can all be removed as it clashes with Hamish's work to make them add that label in by default, currently showing same label twice

File: data/en/1_0005_updated.json [hunk 14]
```diff
@@ -0,0 +1,2172 @@
...
+            "id": "introduction-group",
+            "title": "Introduction"
+        },
+        {
+            "blocks": [{
+                    "id": "pay-pattern-frequency",
+                    "questions": [{
+                        "answers": [{
+                            "id": "pay-pattern-frequency-answer",
+                            "label": "Please select all that apply. You will be asked to provide information for each of the selected pay patterns.",
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: Remove "Please select all that apply." from label as this text is added by default for checkboxes now

File: data/en/1_0005_updated.json (hunk 11 above)
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: This says fortnightly not four weekly

Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: is there a double space between to and the

Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: double space between to and the

File: data/en/1_0005_updated.json [hunk 15]
```diff
@@ -0,0 +1,2172 @@
...
+                                    "label": "other",
+                                    "q_code": "97w",
+                                    "value": "other"
+                                }
+                            ],
+                            "type": "Checkbox"
+
+                        }],
+                        "id": "weekly-pay-significant-changes-options-question",
+                        "title": "What were the reasons for the changes to  the <em>total pay or number</em> of <em>weekly</em> paid employees?",
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: double space between to and the

File: data/en/1_0005_updated.json [hunk 16]
```diff
@@ -0,0 +1,2172 @@
...
+                                {
+                                    "label": "other",
+                                    "q_code": "97f",
+                                    "value": "other"
+                                }
+                            ],
+                            "type": "Checkbox"
+                        }],
+                        "id": "fortnightly-pay-significant-changes-options-question",
+                        "title": "What were the reasons for the changes to  the <em>total pay or number</em> of <em>fortnightly</em> paid employees?",
```
Comment (charlotte-hirst) [inline data/en/1_0005_updated.json]: double space between to and the

**Review by charlotte-hirst — CHANGES_REQUESTED** [review]

**Review by yoganandkunche — CHANGES_REQUESTED** [review]
@MIslam92 Missing check box option 'fewer temporary staff' on summary screen even when selected.

screen shot 2018-01-26 at 12 41 20

File: data/en/1_0005_updated.json [hunk 17]
```diff
@@ -0,0 +1,2169 @@
...
+                            "We will treat your data securely and confidentially."
+                        ]
+                    }]
+                }],
+                "preview_content": {
+                    "id": "preview",
+                    "title": "Information you need",
+                    "content": [{
+                        "title": "Employee pay frequency",
+                        "description": "You will need to indicate how frequently employees are paid. For example weekly, fortnightly, calendar monthly, four weekly or five weekly."
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: This text doesn't match spec file under `Employee pay frequency`

File: data/en/1_0005_updated.json [hunk 18]
```diff
@@ -0,0 +1,2169 @@
...
+                                "Any redundancy pay and pay in lieu of notice payments",
+                                "Trainees on government schemes",
+                                "Director’s fees",
+                                "Employer’s NI and contribution to pension schemes",
+                                "Employees working abroad unless paid directly from this business’s GB payroll",
+                                "Signing on fees (sporting professionals)",
+                                "Payment in lieu of notice",
+                                "Accrued holiday pay",
+                                "Employees in Northern Ireland",
+                                "Benefits employees receive through pay, for example, family working tax credit",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: think this should be `received`

File: data/en/1_0005_updated.json [hunk 19]
```diff
@@ -0,0 +1,2169 @@
...
+                                        "overtime and shift allowance payments",
+                                        "advanced holiday pay",
+                                        "pay award arrears",
+                                        "voluntary employee contributions to, and annual profit from, profit related pay schemes"
+                                    ]
+                                },
+                                {
+                                    "title": "Exclude:",
+                                    "list": [
+                                        "any redundancy pay and pay in lieu of notice payments",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: missing `(taxable and non-taxable)` from spec

File: data/en/1_0005_updated.json [hunk 20]
```diff
@@ -0,0 +1,2169 @@
...
+                            "q_code": "300w",
+                            "type": "TextArea",
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_TEXTAREA": "Enter a comment to continue."
+                                }
+                            }
+                        }],
+                        "id": "weekly-pay-significant-changes-other-specify-question",
+                        "title": "Please describe the changes to the <em>total pay or number</em> of <em>weekly</em> paid employees:",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: Should be `What were' and phrased as a question

File: data/en/1_0005_updated.json [hunk 21]
```diff
@@ -0,0 +1,2169 @@
...
+                                        "overtime and shift allowance payments",
+                                        "advanced holiday pay",
+                                        "pay award arrears",
+                                        "voluntary employee contributions to, and annual profit from, profit related pay schemes"
+                                    ]
+                                },
+                                {
+                                    "title": "Exclude:",
+                                    "list": [
+                                        "any redundancy pay and pay in lieu of notice payments",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: missing `(taxable and non-taxable)`

File: data/en/1_0005_updated.json [hunk 22]
```diff
@@ -0,0 +1,2169 @@
...
+                        "title": "What was the <em>total gross calendar monthly pay</em> paid to employees in {{exercise.period_str}}?",
+                        "type": "General",
+                        "guidance": {
+                            "content": [{
+                                    "title": "Include:",
+                                    "list": [
+                                        "bonuses or commissions",
+                                        "overtime and shift allowance payments",
+                                        "advanced holiday pay",
+                                        "pay award arrears",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: no `advanced holiday pay` and just `pay awards` as in spec

File: data/en/1_0005_updated.json [hunk 23]
```diff
@@ -0,0 +1,2169 @@
...
+                                    "list": [
+                                        "any redundancy pay and pay in lieu of notice payments (taxable and non-taxable)",
+                                        "trainees on government schemes",
+                                        "director’s fees",
+                                        "employer’s NI and contribution to pension schemes",
+                                        "employees working abroad unless paid directly from this business’s GB payroll",
+                                        "signing on fees (sporting professionals)",
+                                        "accrued holiday pay",
+                                        "employees in Northern Ireland",
+                                        "expenses payments for attending meetings, for example, councillors"
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: `or training courses` after attending meetings

File: data/en/1_0005_updated.json [hunk 24]
```diff
@@ -0,0 +1,2169 @@
...
+                                "decimal_places": 2,
+                                "max_value": {
+                                    "answer_id": "calendar-monthly-pay-gross-pay-answer",
+                                    "exclusive": true
+                                }
+                            }
+                        ],
+                        "id": "calendar-monthly-pay-breakdown-question",
+                        "title": "Of the <em>{{answers.monthly_total_gross|format_currency}}</em> paid to <em>calendar monthly</em> paid employees in {{exercise.period_str}}, what other amounts were paid?:",
+                        "type": "General",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: remove colon

File: data/en/1_0005_updated.json [hunk 25]
```diff
@@ -0,0 +1,2169 @@
...
+                                }
+                            ],
+                            "q_code": "190m",
+                            "type": "Radio"
+                        }],
+                        "id": "calendar-monthly-pay-significant-changes-paid-employees-question",
+                        "title": "Did any significant changes occur to the <em>total pay or number</em> of <em>calendar monthly</em> paid employees?",
+                        "type": "General",
+                        "description": "<p>This is your own interpretation of whether there has been a significant change in  {{respondent.trad_as_or_ru_name}}’s figures compared with the previous month.</p><p>Also consider whether there has been a significant change from the same month in the previous year.</p>",
+                        "guidance": {
```
Comment (shahi645) [inline data/en/1_0005_updated.json:1204]: missing a `Please` before also

File: data/en/1_0005_updated.json [hunk 26]
```diff
@@ -0,0 +1,2169 @@
...
+                                "block": "Four Weekly Pay",
+                                "when": [{
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "condition": "equals",
+                                    "value": "No"
+                                }]
+                            }
+                        }
+                    ]
+
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: Should be `Four Weekly Pay`

Comment (shahi645) [inline data/en/1_0005_updated.json]: remove label

Comment (shahi645) [inline data/en/1_0005_updated.json]: `?` instead of `:`

Comment (shahi645) [inline data/en/1_0005_updated.json]: needs advanced holiday pay

Comment (shahi645) [inline data/en/1_0005_updated.json]: or training courses

Comment (shahi645) [inline data/en/1_0005_updated.json]: remove colon

Comment (shahi645) [inline data/en/1_0005_updated.json]: remove label

Comment (shahi645) [inline data/en/1_0005_updated.json]: should be `five weekly`

Comment (shahi645) [inline data/en/1_0005_updated.json]: doesnt match spec

**Review by shahi645 — CHANGES_REQUESTED** [review]

File: data/en/1_0005_updated.json [hunk 27]
```diff
@@ -645,7 +645,7 @@
                             }
                         }],
                         "id": "weekly-pay-significant-changes-other-specify-question",
-                        "title": "Please describe the changes to the <em>total pay or number</em> of <em>weekly</em> paid employees:",
+                        "title": "What were the changes to the <em>total pay or number</em> of <em>weekly</em> paid employees:",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: needs to b e a `?` at the end

File: data/en/1_0005_updated.json [hunk 28]
```diff
@@ -2084,7 +2083,7 @@
                             }
                         }],
                         "id": "five-weekly-pay-significant-changes-other-specify-question",
-                        "title": "Please describe the changes to the <em>total pay or number</em> of <em>five weekly paid</em> employees:",
+                        "title": "What were the changes to the <em>total pay or number</em> of <em>five weekly paid</em> employees:",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: `?`

File: data/en/1_0005_updated.json [hunk 29]
```diff
@@ -0,0 +1,2168 @@
...
+                        }, {
+                            "description": "Exclude:",
+                            "list": [
+                                "Any redundancy pay and pay in lieu of notice payments",
+                                "Trainees on government schemes",
+                                "Director’s fees",
+                                "Employer’s NI and contribution to pension schemes",
+                                "Employees working abroad unless paid directly from this business’s GB payroll",
+                                "Signing on fees (sporting professionals)",
+                                "Payment in lieu of notice",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: this line isn't on the spec file

File: data/en/1_0005_updated.json [hunk 30]
```diff
@@ -0,0 +1,2168 @@
...
+                                    "value": "more overtime"
+                                },
+                                {
+                                    "label": "less overtime",
+                                    "q_code": "94w2",
+                                    "value": "less overtime"
+                                },
+                                {
+                                    "label": "pay increase",
+                                    "q_code": "95",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: q_code should be `95w`

File: data/en/1_0005_updated.json [hunk 31]
```diff
@@ -0,0 +1,2168 @@
...
+                            "q_code": "300w",
+                            "type": "TextArea",
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_TEXTAREA": "Enter a comment to continue."
+                                }
+                            }
+                        }],
+                        "id": "weekly-pay-significant-changes-other-specify-question",
+                        "title": "What were the changes to the <em>total pay or number</em> of <em>weekly</em> paid employees:",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: `?` at end not `:`

File: data/en/1_0005_updated.json [hunk 32]
```diff
@@ -0,0 +1,2168 @@
...
+                            "q_code": "300f",
+                            "type": "TextArea",
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_TEXTAREA": "Enter a comment to continue."
+                                }
+                            }
+                        }],
+                        "id": "fortnightly-pay-significant-changes-other-specify-question",
+                        "title": "Please describe the changes to the <em>total pay or number</em> of <em>fortnightly</em> paid employees:",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: needs to be question as in spec

File: data/en/1_0005_updated.json [hunk 33]
```diff
@@ -0,0 +1,2168 @@
...
+                                    "value": "No"
+                                }
+                            ],
+                            "q_code": "190m",
+                            "type": "Radio"
+                        }],
+                        "id": "calendar-monthly-pay-significant-changes-paid-employees-question",
+                        "title": "Did any significant changes occur to the <em>total pay or number</em> of <em>calendar monthly</em> paid employees?",
+                        "type": "General",
+                        "description": "<p>This is your own interpretation of whether there has been a significant change in  {{respondent.trad_as_or_ru_name}}’s figures compared with the previous month.</p><p>Please, also consider whether there has been a significant change from the same month in the previous year.</p>",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: remove `,` after please

File: data/en/1_0005_updated.json [hunk 34]
```diff
@@ -0,0 +1,2168 @@
...
+                                "block": "Four Weekly Pay",
+                                "when": [{
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "condition": "equals",
+                                    "value": "No"
+                                }]
+                            }
+                        }
+                    ]
+
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: Should be question as in spec

Comment (shahi645) [inline data/en/1_0005_updated.json]: Should be `pay awards`

Comment (shahi645) [inline data/en/1_0005_updated.json]: `?` mot `:`

Comment (shahi645) [inline data/en/1_0005_updated.json]: missing `advanced holiday pay`

Comment (shahi645) [inline data/en/1_0005_updated.json]: remove label

Comment (shahi645) [inline data/en/1_0005_updated.json]: `?` not `:`

**Review by shahi645 — CHANGES_REQUESTED** [review]

File: data/en/1_0005_updated.json [hunk 35]
```diff
@@ -0,0 +1,2166 @@
...
+                            "q_code": "300f",
+                            "type": "TextArea",
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_TEXTAREA": "Enter a comment to continue."
+                                }
+                            }
+                        }],
+                        "id": "fortnightly-pay-significant-changes-other-specify-question",
+                        "title": "Please describe the changes to the <em>total pay or number</em> of <em>fortnightly</em> paid employees?",
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: Does not match spec

File: data/en/1_0005_updated.json [hunk 36]
```diff
@@ -0,0 +1,2166 @@
...
+                                    "id": "calendar-monthly-pay-significant-changes-paid-employees-answer",
+                                    "condition": "equals",
+                                    "value": "No"
+                                }]
+                            }
+                        }
+                    ]
+                },
+                {
+                    "id
```
Comment (shahi645) [inline data/en/1_0005_updated.json]: Doesn't match spec

Comment (shahi645) [inline data/en/1_0005_updated.json]: doesnt match spec needs a `?`

Comment (shahi645) [inline data/en/1_0005_updated.json]: `?`

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Review by shahi645 — APPROVED** [review]
👍

File: data/en/1_0005.json [hunk 37]
```diff
@@ -398,6 +399,11 @@
                             "min_value": {
                                 "value": 0,
                                 "exclusive": true
+                            },
+                            "validation": {
+                                "messages": {
+                                    "MANDATORY_NUMBER": "Enter an answer to continue."
```
Comment (jonnyshaw89) [inline data/en/1_0005.json]: Why are we changing this schema?
Also this value is the same as the default

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by charlotte-hirst — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by yoganandkunche** [comment]
@MIslam92 'fewer temporary staff' still missing on the summary page.

**Review by yoganandkunche — APPROVED** [review]
LGTM 
image

**Pull request merged**